BootloaderCommonPkg/Library/IppCryptoLib/rsa_verify.c

```diff
@@ -91,7 +98,18 @@ int VerifyRsaSignature (const void *hash, const void *rsa_n,  const void *rsa_e,
   }
 
   signature_verified = 0;
-  err = ippsRSAVerifyHash_PKCS1v15_rmf((const Ipp8u *)hash, (Ipp8u *)sig, &signature_verified, rsa_key_s, ippsHashMethod_SHA256 (), scratch_buf);
+  if (SignatureHdr->HashAlg == HASH_TYPE_SHA256){

```

Comment:
Here, you need to add PcdIppHashLibSupportedMask check to save size.

Reply:
Thanks, i would add.